docker-compose.yml

```diff
@@ -533,6 +543,20 @@ services:
       - sentry-vroom:/var/vroom/sentry-profiles
     profiles:
       - feature-complete
+  uptime-checker:
+    <<: *restart_policy
+    image: "$UPTIME_CHECKER_IMAGE"
+    command: run
+    environment:
+      UPTIME_CHECKER_RESULTS_KAFKA_CLUSTER: kafka:9092
+      UPTIME_CHECKER_REDIS_HOST: redis://redis:6379
```

Comment:
Should we also expose `UPTIME_CHECKER_ALLOW_INTERNAL_IPS` and `UPTIME_CHECKER_HTTP_CHECKER_DNS_NAMESERVERS` here? That being commented out is good enough for future references.

Reply:
```suggestion
    environment:
      UPTIME_CHECKER_RESULTS_KAFKA_CLUSTER: kafka:9092
      UPTIME_CHECKER_REDIS_HOST: redis://redis:6379
      # Set to `true` will allow uptime checks against private IP addresses
      UPTIME_CHECKER_ALLOW_INTERNAL_IPS: false
      # DNS name servers to use when making checks in the http checker. Separated by commas.
      # UPTIME_CHECKER_HTTP_CHECKER_DNS_NAMESERVERS: "8.8.8.8,8.8.4.4"
      # The number of times to retry failed checks before reporting them as failed
      UPTIME_CHECKER_FAILURE_RETRIES: "0"
```

Reply:
Yeah that sounds good